docs/.vuepress/theme/components/Versions.vue

```diff
@@ -66,42 +66,3 @@ export default {
   }
 };
 </script>
-
-<style lang="stylus">
-.nav-versions
-  margin-right 2rem
```

Comment:
Can I ask why have we moved the styles from components to identical named files in some arbitrary location? Quite old school approach in an app that is build with components

Reply:
The worst thing about styling in component files is that in component files we can't use variables from other stylesheets than the `palette.styl`. That causes that the theming was impossible in the component files. Dark theme colors are in the `theme-dark-palette.styl` but we don't have access to these variables in the components.

Reply:
We can't import it? Or if `palette.styl` is available then import `theme-dark-palette.styl` in `palette-style`?